appveyor.yml

```diff
@@ -1,28 +0,0 @@
-image: Visual Studio 2015
-
-environment:
-  NODEJS_VERSION: "8"
```

Comment:
This fails with the following error:

```
C:\projects\tree-sitter-bash\node_modules\tree-sitter-cli\install.js:68
  const requestUrl = new URL(url)
                     ^
ReferenceError: URL is not defined
```

since this is testing an old Node version (EOL'd [in 2020](https://github.com/nodejs/Release#end-of-life-releases)) and I don't see what it's doing that the GitHub Actions CI is doing, I "fixed the issue".